### PR TITLE
Add a Delayed Job web interface

### PR DESCRIPTION
**Pull request by LS80**
Uses https://github.com/ejschmitt/delayed_job_web to enable better visibility of number of enqueued jobs, failed jobs etc.

Will be at https://shipment-tracker.fundingcircle.co.uk/delayed_job/overview

Secured via Auth0 (session expires after 24 hours).

### CODE REVIEW

**Comment by LS80** [comment]
It is not secured. I'll see how easy it is to add it through `omniauth-auth0`.

File: config/initializers/omniauth.rb [hunk 1]
```diff
@@ -5,17 +5,22 @@
 Rails.configuration.login_url          = '/auth/auth0'
 Rails.configuration.login_callback_url = '/auth/auth0/callback'
 
+auth0_provider_args = {
+  client_id:     ENV.fetch('AUTH0_CLIENT_ID', nil),
+  client_secret: ENV.fetch('AUTH0_CLIENT_SECRET', nil),
+  namespace:     ENV.fetch('AUTH0_DOMAIN', 'fundingcircle.auth0.com'),
```
Comment (sgerrand) [inline config/initializers/omniauth.rb]: 🙋 Please don't set a default value for this environment variable.

File: config/initializers/omniauth.rb [hunk 2]
```diff
@@ -5,17 +5,22 @@
...
+  connection:    ENV.fetch('AUTH0_CONNECTION', 'Username-Password-Authentication'),
```
Comment (sgerrand) [inline config/initializers/omniauth.rb]: Suggest not setting a default for this either.

File: config/initializers/omniauth.rb (hunk 1 above)
Comment (LS80) [reply config/initializers/omniauth.rb]: I didn't change any of the defaults. The `omniauth` configuration is the same as it was.

Comment (sgerrand) [reply config/initializers/omniauth.rb]: 🙇 Will resolve separately.

**Pull request merged**